### ISSUE TITLE
Predict species of iris flower using K Nearest Neighbors model

### ISSUE THREAD

**Issue by ShashankP19** [WebClub-NITK/Hacktoberfest-2k18#12]
### Description
Given the dataset on iris flowers, predict the species of flower given the following features : 
1. sepal length
2. sepal width
3. petal-length 
4. petal-width

There are three species:
1. virginica
2. setosa
3. versicolor

Use K Nearest Neighbors classifier
The dataset can be found ![here](https://archive.ics.uci.edu/ml/datasets/iris)

### Details
- Technical Specifications: This is a beginner level machine learning problem.
- Type of issue: Single issue
- Time Limit: You have 3 days to submit the solution for this issue. 

### Issue requirements / progress
1. Use K Nearest Neighbors classifier model. Train the model on train data. 
2. Predict target values using test data
3. Find accuracy of the model comparing with actual test data targets

### Resources
- To help you get started, the starter code can be found ![here](https://drive.google.com/drive/folders/1sNs_F9ihxQgk8OCMu9wE-SE9Y1gsoM27?usp=sharing)
- scikit-learn ![KNeighborsClassifier](http://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html)
- pandas ![tutorial](https://www.tutorialspoint.com/python_pandas/)
- numpy ![tutorial](https://www.tutorialspoint.com/numpy)

### Directory Structure
Place your solution file in path as follows.  
`ML --->Iris ---> KNN ---> <your_name> ---><solution_file>`
For example, `ML ---> Iris ---> KNN ---> Shashank ---> iris_knn.ipynb`
Create necessary folders.

**Comment by mehnazyunus** [comment]
Hi, I'd like to work on Gaussian Naive Bayes Classifier.

**Comment by ShashankP19** [comment]
Hi @mehnazyunus , you are assigned to work on Gaussian Naive Bayes Classifier. Go ahead.

**Comment by shakeelsamsu** [comment]
Hi, I'd like to work on the Support Vector Classifier

**Comment by GajeshS** [comment]
Hi, I would like to work on the MLP Implementation

**Comment by bhuvanakundumani** [comment]
I would like to work on Random Forest Classifier.

**Comment by bhuvanakundumani** [comment]
Hi 

I would like to work on Decision Tree Calssifier

**Comment by ShashankP19** [comment]
@bhuvanakundumani You are assigned Decision Tree Classifier. Go ahead.

**Comment by bhuvanakundumani** [comment]
Ok. Thanks

**Comment by bhuvanakundumani** [comment]
Hi,
Implemented the Decision tree classifier

**Comment by Madhuparna04** [comment]
Hi ,I would like to work on KNN classifier

**Comment by ShashankP19** [comment]
@Madhuparna04  KNN classifier has already been taken. Choose some other classifier.
